server/config/config.js

```diff
@@ -4,7 +4,8 @@ const dbConfig = {
   database: process.env.POSTGRES_DB,
   host: process.env.POSTGRES_HOST,
   port: process.env.POSTGRES_PORT,
-  dialect: 'postgres'
+  dialect: 'postgres',
+  logging: false

```

Comment:
Should logging be controlled by an environment variable? It would be useful to turn on in dev.

Reply:
Changed so that it uses an environment variable now